src/script/components/MessagesList/MessageList.tsx

```diff
@@ -268,15 +271,26 @@ export const MessagesList: FC<MessagesListParams> = ({
     }
   };
 
+  const setDeliveredIndicatorWidth = (element: HTMLDivElement) => {
+    if (element?.offsetWidth > 0) {
+      setRightMarginWidth(element?.offsetWidth);
+    }
+  };
+
   return (
     <>
+      {rightMarginWidth === 0 && <DeliveredIndicator ref={setDeliveredIndicatorWidth} isLastDeliveredMessage={false} />}
```

Comment:
If we agree on this approach, it can be modified to 
```
if(rightMarginWidth === 0) return <DeliveredIndicator ref={setDeliveredIndicatorWidth} isLastDeliveredMessage={false} />

return ( ...
```
to avoid one extra render. Then the `MessageList.test` must be adjusted to ignore the first render